pkg/agent/sysadvisor/plugin/qosaware/resource/cpu/region/region_dedicated_numa_exclusive.go

```diff
@@ -38,6 +42,7 @@ import (
 
 type QoSRegionDedicatedNumaExclusive struct {
 	*QoSRegionBase
+	podUID string
```

Comment:
personally, I would prefer not to define podUID here, since it indicates that only contain one pod can be supported, but the podSet indicates that we should support multiple pods, and it may cause a lot of confusion as the logic goes more and more complex

Reply:
or we can redefine the high-level design, i.e. "whether it is reasonable to make every region to contain a podSet "

Reply:
It is necessary to keep the podset because we need to calculate resource usage by it. NewQoSRegionDedicatedNumaExclusive accept containerInfo as one of the parameters indicate that only this region only belong to single Pod.

Reply:
still confused for keeping both podSet and podUID; besides, the reason for `NewQoSRegionDedicatedNumaExclusive accept containerInfo as one of the parameters` is not a solid reason for keeping podUID or indicating there should be only one pod in this region, since all regions accept containerInfo as one of the parameters (cause we need for its owner pool name) ...

Reply:
well, I removed podUID of QoSRegionDedicatedNumaExclusive and use getPodUID to get it.